05_syntax_lab/01.py

```diff
@@ -1,5 +1,11 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+print "Enter Number"
+big = int(raw_input())
+
+for i in range(1,10) :
+    print "Enter Number"
+    num = int(raw_input())
+    big = max(num,big)
+
+print "Max Number is %d",big
+
+    
```

Comment:
Within the loop you can recognize if it is in the first iteration and only assign ```big = num```